pkg/vcs/linux.go

```diff
@@ -257,10 +257,12 @@ func linuxAlterConfigs(cf *kconfig.ConfigFile, tags map[string]bool, dt debugtra
 		// UBSAN is broken in multiple ways before v5.3, see:
 		// https://github.com/google/syzkaller/issues/1523#issuecomment-696514105
 		"UBSAN": "v5.3",
-		// First, we disable coverage in pkg/bisect because it fails machine testing starting from 4.7.
+		// First, we want to disable coverage in pkg/bisect because it fails machine testing starting from 4.7.
 		// Second, at 6689da155bdcd17abfe4d3a8b1e245d9ed4b5f2c CONFIG_KCOV selects CONFIG_GCC_PLUGIN_SANCOV
 		// (why?), which is build broken for hundreds of revisions.
-		"KCOV": disableAlways,
+		// Still, let's keep it enabled for newer Linux versions -- it keeps the kernel as close to the one
```

Comment:
The other changes that dropped configs did extra runs to verify that the changed kernel still works. With `KCOV`, I thought it would be not really worth extra 20 minutes of bisection time. It's not a sanitizer and it's usually not broken (at least I can't recollect any recent cases), but it might still keep that extra bit of slowness to the kernel that it had when we were fuzzing it and that helped trigger the bug in the first place.

Reply:
But KCOV also leads to increased compilation time, and removing sanitizers also leads to changed timings (potentially even more).
If additional steps is the problem: can we batch testing? say, remove both sanitizers and KCOV together and do a single test. In the expected case (the sanitizer is indeed not needed and the bug is not too timing dependent), we can get away with 1 testing attempt.

Reply:
> But KCOV also leads to increased compilation time

FWIW on a 16 core GCP VM I didn't notice any significant difference.

> say, remove both sanitizers and KCOV together and do a single test.

Yeah, that might make sense to try to drop it together with sanitizers.

Reply:
> FWIW on a 16 core GCP VM I didn't notice any significant difference.

Do you see a diff for, say, UBSAN?
If there increase in CPU time? Some users may have even smaller machines (e.g. my laptop :)).
Also there may be diff in vmlinux size for asset downloads (esp if we do it for reproducers).